src/Accordion.php

```diff
@@ -154,6 +155,20 @@ public function renderItems()
         return implode("\n", $items);
     }
 
+    /**
+     * check wether accordion is expanded
+     * This method will check if expand properties exist and determine value of this properties
+     * 
+     * @param array $item
+     * @return bool
+     */
+    private function _isAccordionExpanded($item){
+        if(isset($item['expand'])){
```

Comment:
There should be spaces after `if` statement and between `)` and `{` according [Yii code styles](https://github.com/yiisoft/yii2/blob/master/docs/internals/core-code-style.md#54-control-statements).